examples/cpp/content_filter/PublisherApp.hpp

```diff
@@ -17,8 +17,8 @@
  *
  */
 
-#ifndef _FASTDDS_CONTENT_FILTER_PUBLISHER_APP_HPP_
-#define _FASTDDS_CONTENT_FILTER_PUBLISHER_APP_HPP_
+#ifndef FASTDDS_EXAMPLES_CPP_CONTENT_FILTER__PUBLISHER_APP_HPP
```

Comment:
Apply for the rest of the file
```suggestion
#ifndef FASTDDS_EXAMPLES_CPP_CONTENT_FILTER__PUBLISHERAPP_HPP
```